api/kessel/inventory/v1beta1/resources/notifications_integrations_service.proto

```diff
@@ -58,4 +74,11 @@ service KesselNotificationsIntegrationService {
       body : "*"
     };
   };
+  
+  rpc ListNotificationsIntegrations(ListNotificationsIntegrationsRequest) returns (stream ListNotificationsIntegrationsResponse) {
+    option (google.api.http) = {
+      get : "/api/inventory/v1beta1/resources/notifications-integrations"
+      body : "*"
```

Comment:
nitpick: IMO we shouldn't support a body for get requests since (while allowed) it is not recommended per http spec and could cause issues with some clients

Reply:
Thanks for the callout, didn't mean for this to have a http body 👍